src/CMakeLists.txt

```diff
@@ -34,31 +29,31 @@ IF(QWT_MAJOR_VERSION LESS 6)
 	MESSAGE(STATUS "QWT6 is required.")
 ENDIF(QWT_MAJOR_VERSION LESS 6)
 
-IF(QT4_FOUND AND QWT_FOUND AND QWT_MAJOR_VERSION EQUAL 6)
-  QT4_WRAP_CPP(lineplotwraps ../include/srsgui/common/Lineplot.h)
-  QT4_WRAP_CPP(pointplotwraps ../include/srsgui/common/Pointplot.h)
-  QT4_WRAP_CPP(spectrogramplotwraps ../include/srsgui/common/Spectrogramplot.h)
-  QT4_WRAP_CPP(complex complexplot/ComplexWidget.h complexplot/ComplexplotWrapper.h)
-  QT4_WRAP_CPP(real realplot/RealWidget.h realplot/RealplotWrapper.h)
-  QT4_WRAP_CPP(scatter scatterplot/ScatterWidget.h scatterplot/ScatterplotWrapper.h)
-  QT4_WRAP_CPP(waterfall waterfallplot/WaterfallWidget.h waterfallplot/WaterfallplotWrapper.h)
-  QT4_WRAP_CPP(textedit textedit/TextEditWidget.h textedit/TextEditWrapper.h ../include/srsgui/plot/text_edit.h)
-  QT4_WRAP_CPP(keyvalue keyvalue/KeyValueWidget.h keyvalue/KeyValueWrapper.h ../include/srsgui/plot/key_value.h)
+IF(Qt5_FOUND AND QWT_FOUND AND QWT_MAJOR_VERSION EQUAL 6)
+  QT5_WRAP_CPP(lineplotwraps ../include/srsgui/common/Lineplot.h)
+  QT5_WRAP_CPP(pointplotwraps ../include/srsgui/common/Pointplot.h)
+  QT5_WRAP_CPP(spectrogramplotwraps ../include/srsgui/common/Spectrogramplot.h)
+  QT5_WRAP_CPP(complex complexplot/ComplexWidget.h complexplot/ComplexplotWrapper.h)
+  QT5_WRAP_CPP(real realplot/RealWidget.h realplot/RealplotWrapper.h)
+  QT5_WRAP_CPP(scatter scatterplot/ScatterWidget.h scatterplot/ScatterplotWrapper.h)
+  QT5_WRAP_CPP(waterfall waterfallplot/WaterfallWidget.h waterfallplot/WaterfallplotWrapper.h)
+  QT5_WRAP_CPP(textedit textedit/TextEditWidget.h textedit/TextEditWrapper.h ../include/srsgui/plot/text_edit.h)
```

Comment:
not sure why I added the `text_edit.h` back then but CMake showed me
```
[  3%] Generating __/include/srsgui/common/moc_Spectrogramplot.cpp
[  5%] Generating __/include/srsgui/common/moc_Lineplot.cpp
/home/anpu/src/srsGUI/include/srsgui/plot/key_value.h:0: Note: No relevant classes found. No output generated.
[  6%] Generating complexplot/moc_ComplexWidget.cpp
[  8%] Generating complexplot/moc_ComplexplotWrapper.cpp
[  8%] Generating 
```

So I removed them in this line and it still compiles

Reply:
Sure, I will check it, would be nice if we could trim any unneeded headers. The weird thing is that I did not get that warning.
About the warning, probably it is issued because the moc compiler does not find any work to do in that header file.

Reply:
i've removed those header files.